data/clients.json

```diff
@@ -631,6 +631,15 @@
         ],
         "url": "https://trillian.im"
     },
+    {
+        "doap": null,
+        "last_renewed": null,
```

Comment:
If it does not have a date It will not be listed if I'm not mistaken, since the point of this list is to keep it up to date.
Can you add a date, please?

Thanks in advance @Echolon

Reply:
Done